core/builder.js

```diff
@@ -3,10 +3,10 @@
     Handles page building including defer blocks (fragment loading),
     rendering, in-place pagination.
 */
-define('builder',
-    ['log', 'jquery', 'templates', 'models', 'requests', 'settings', 'z',
-     'nunjucks.compat'],
-    function(log, $, nunjucks, models, requests, settings, z) {
+define('core/builder',
+    ['core/log', 'core/nunjucks', 'core/models', 'core/requests', 'core/settings', 'core/z',
+     'core/nunjucks.compat', 'jquery'],
+    function(log, nunjucks, models, requests, settings, z, nunjucksCompat, $) {
```

Comment:
i like adding line breaks to the dependency injection line/function arguments to match the line breaks in the dependency list. Makes it easier to match up when removing/adding modules.

```
['core/log', ... 'core/z',
 'core/nunjucks.compat', 'jquery']
function(log, ... z, 
         nunjucksCompat, $)
```